kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateList.js

```diff
@@ -265,20 +351,34 @@ define([
             });
         }
 
+        /**
+         * Update the table with a new jobState object
+         * @param {object} jobState
+         */
+        function updateJobStatusInTable(jobState) {
+            // select the appropriate row
+            dataTable
+                .DataTable()
+                .row('#job_' + jobState.job_id)
+                .data(jobState)
+                .draw();
+        }
+
         /**
          * parse and update the row with job info
          * @param {object} _ - job manager context
          * @param {object} message
          */
         function handleJobInfo(_, message) {
-            if (message.jobId && message.jobInfo && widgetsById[message.jobId]) {
-                widgetsById[message.jobId].updateParams(message.jobInfo);
-                jobManager.removeListener(message.jobId, 'job-info');
-                jobManager.model.setItem(
-                    `exec.jobs.params.${message.jobId}`,
-                    message.jobInfo.job_params[0]
-                );
-            }
+            const jobId = message.jobId;
+            jobManager.model.setItem(`exec.jobs.params.${jobId}`, message.jobInfo.job_params[0]);
+            jobManager.removeListener(jobId, 'job-info');
+            // update the table
+            dataTable
+                .DataTable()
+                .row('#job_' + jobId)
+                .invalidate()
+                .draw();
```

Comment:
and another simple update function!

`invalidate()` gets rid of cached data so that the job info is redrawn